### PR TITLE
Jonah wip comments

### PR DESCRIPTION
**Pull request by jsolem**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
```
Comment (tremerent) [inline .vscode/launch.json]: Pls add `.vscode/` to the gitignore.

File: Models/Artefact.cs [hunk 2]
```diff
@@ -13,15 +13,14 @@ public class Artefact
...
         [JsonRequired]
         public string Description { get; set; }
         public System.DateTime CreatedAt { get; set; }
+
         public IEnumerable<ArtefactCategory> CategoryJoin { get; set; }
-        // [InverseProperty("Artefact")]
         public IEnumerable<ArtefactDocument> Images { get; set; }
-        // [InverseProperty("Artefact")]
-        // public ICollection<ArtefactDocument> Documents { get; set; }
+        public IEnumerable<ArtefactComment> Comments { get; set; }
```
Comment (tremerent) [inline Models/Artefact.cs]: May be one of those 'misunderstanding dotnet' moments but does this mean we'll get all comments on an artefact every time we load that artefact from the DB? 
That's bad.

Comment (jsolem) [reply Models/Artefact.cs]: No it doesn't - to load all comments, would need to do a -
`_context.Artefacts.Include(artefact => artefact.Comments)`

https://www.learnentityframeworkcore.com/lazy-loading

I don't think we have lazy-loading enabled at the moment, but it's just a property in `ApplicationDbContext`

Comment (tremerent) [reply Models/Artefact.cs]: Okay, misunderstanding it is.

**Pull request merged**